src/main/java/org/jabref/gui/groups/GroupTreeController.java

```diff
@@ -87,7 +90,13 @@ public void initialize() {
                 this::updateSelection
         );
 
-        viewModel.filterTextProperty().bind(searchField.textProperty());
+        // We try to to prevent publishing changes in the searchfield directly to the search task that takes some time
+        // for larger group structures.
+        final Timer searchTask = FxTimer.create(Duration.ofMillis(400), () -> {
```

Comment:
I think, it makes sense to extract this code to a helper method in `BindingsHelper`.